src/world/scheduler/builder.rs

```diff
@@ -262,6 +262,29 @@ impl WorkloadBuilder {
             Err(err) => panic!("{:?}", err),
         }
     }
+    /// Extends the builder with the following [`WorkloadBuilder`].
+    /// This allows us to collect systems in different builders before joining them together.
+    ///
+    /// [`WorldloadBuilder`]: struct.WorldBuilder.html
```

Comment:
Whoops 👍 
It would fun to build a simple linter to help catch mistakes like this. I feel like you could use the existing `target/doc` to check the html filenames and contents for anchor tags.